src/main/java/lotto/domain/WinningLottery.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public enum WinningLottery {
+  FIRST(6, 2000000000),
```

Comment:
자바에선 이렇게 큰 숫자 단위에 대해서는 콤마(`,`)대신 언더스코어(`_`)를 제공해요 ㅎㅎ
```suggestion
  FIRST(6, 2_000_000_000),
```
이렇게 좀 더 가독성을 높힐 수 있습니다 ㅎ